crux-test/test/crux/examples_test.clj

```diff
@@ -32,17 +32,23 @@
     (t/is (= #{[:dbpedia.resource/Pablo-Picasso]} (ex/example-query node)))
     (t/is (not (empty? (ex/example-query-valid-time node))))
 
-    ;; Testing http-server/http-client using the standalone node
-    (with-open [http-server ^Closeable (ex/example-start-http-server node)
-                remote-api ^Closeable (ex/example-start-http-client)]
+    ;; Testing example standalone node is closed properly
+    (t/is (nil? (ex/example-close-node node)))))
+
+(t/deftest test-example-http-node
+  (with-open [underlying-node ^Closeable (ex/example-start-standalone-http)
+              node ^Closeable (ex/example-start-http-client)]
+    ;; Testing example standalone node is created properly
```

Comment:
add a t/testing "something" rather than a comment?

Reply:
doh. thanks :smile:

Reply:
ah, this was existing code I moved about - but yeah, will update regardless